vagrant/provision.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+set -x
+
+export DEBIAN_FRONTEND=noninteractive 
+apt-get update
+apt-get -y upgrade
+apt-get -y install apt-transport-https
+
+echo "export EDITOR=vim" >> /home/vagrant/.bashrc
+
+snap install microk8s --classic --channel=1.13/stable
+microk8s.status --wait-ready
+ufw allow in on cbr0
+ufw allow out on cbr0
+ufw default allow routed
+
+microk8s.enable dashboard
+microk8s.enable registry
+microk8s.enable storage
+microk8s.enable dns
+microk8s.enable prometheus
+
+snap install helm --classic --channel=2.16
+helm init --wait
+
+microk8s.kubectl config view --raw > /vagrant/.kube-config
+
+snap alias microk8s.kubectl kubectl
+snap alias microk8s.docker docker
+
+# allow privileged
+echo "--allow-privileged=true" >> /var/snap/microk8s/current/args/kubelet
+echo "--allow-privileged=true" >> /var/snap/microk8s/current/args/kube-apiserver
+systemctl restart snap.microk8s.daemon-kubelet.service
+systemctl restart snap.microk8s.daemon-apiserver.service
+
+# allow connections to outside
+iptables -P FORWARD ACCEPT
+apt-get install --yes iptables-persistent
+
+#snap alias microk8s.helm helm
+helm init --wait
```

Comment:
This will work only for the root user. Please run as a `vagrant` user.

Reply:
it works fine for me, why? 🤔

Reply:
On the other hand
```
    default: + su vagrant -c helm init --wait
    default: bash: helm: command not found
    default: + usermod -a -G microk8s vagrant
    default: usermod: group 'microk8s' does not exist
```

Reply:
AFAIK provisioning is run under root account.
As for those errors it seems microk8s has not been installed correctly.